src/did_resolve.rs

```diff
@@ -1096,6 +1100,84 @@ impl<'a> DIDResolver for SeriesResolver<'a> {
     }
 }
 
+pub(crate) async fn easy_resolve(did: &str, resolver: &dyn DIDResolver) -> Result<Document, Error> {
```

Comment:
I guess this would be replaced by an implementation of the `Try` trait, once it's stabilised. Maybe put a TODO?

Reply:
Added: 0bae94b33e2657e34a4b9cf68bd786db0a958efd